public/html/index.html

```diff
@@ -30,8 +30,15 @@
       crossorigin
       type="font/woff2"
     />
+    <style>
+      .splash-screen {
```

Comment:
Can you add `.splash-screen` into `base.css` instead? 

Then you don't need to repeat it. We should add a comment with the token name for this colour too in case we need to change it

Reply:
> We should add a comment with the token name for this colour too in case we need to change it

hm, don't understand, where we should add this comment?

Reply:
I just meant to add a comment here with the semantic name of the color. Looking at the tokens and it seems to be `lightModeBrown.11` / `darkModeBrown.3`. 

```
 lightModeBrown: {
    11: { value: '#4A423B' },
  },
  darkModeBrown: {
    3: { value: '#4A423B' },
```

Is that the right colour? should it be `background-primary`?

Reply:
hmm, we cannot use semantic tokens for splash screen, as app doesn't know about them when we set splash screen

Reply:
I know we can't use the token here, I'm just saying to add a comment of the token name so we know which colour it's supposed to be:

```
.splash-screen {
 /* lightModeBrown.11  */
 background-color: #4a423b !important;
}
```

Reply:
ahh, I see, ok